injected/integration-test/test-pages/webcompat/pages/modify-cookies.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Modify Cookies</title>
+    <link rel="stylesheet" href="../../shared/style.css">
+    <script>
+        document.cookie = 'keyToBeDeleted="valueToBeDeleted"'
+        document.cookie = 'otherKey="valueToRemain"'
+    </script>
+</head>
+<body>
+    <script src="../../shared/utils.js"></script>
+    <p><a href="../index.html">[WebCompat]</a></p>
+
+    <p>This page verifies that cookie modifications work properly given the <a href="../config/modify-cookies.json">config</a>. At this time, only deletion is supported.</p>
+
+    <script>
+        // Copy from: https://stackoverflow.com/a/2144404
+        function hasCookie(name){
+            return document.cookie.split(';').some(c => {
+                return c.trim().startsWith(name + '=');
+            });
+        }
+
+        // eslint-disable-next-line no-undef
+        test('Only specified cookies should be removed', async () => {
+            const specifiedKey = hasCookie('keyToBeDeleted')
+            const nonexistentKey = hasCookie('nonexistentKey')
+            const otherKey = hasCookie('otherKey')
+
+            return [
+                { name: 'specified cookie entry deleted', result: specifiedKey, expected: false },
+                { name: 'specified cookie entry that is not present on page load', result: nonexistentKey, expected: false },
+                { name: 'other cookie entry untouched', result: otherKey, expected: true }
```

Comment:
Should we add a test that confirms `path` is working as expected?

Reply:
Added test to check both states with path. I'd rather not add a second sub path test to validate some of the other cases (I don't think we need to validate the visibility of the cookie for example, because the behavior of cookie is path not set when deleting won't delete).

The same is true for domain but that's a bit more problematic as we intend to run these in a few places.